presentation-api/controlling-ua/defaultRequest_success-manual.html

```diff
@@ -9,34 +9,52 @@
 
 <script>
     // disable the timeout function for the tests
     // and call 'done()' when the tests cases are finished.
     setup({explicit_timeout: true});
 
-    // -------------------
-    // defaultRequest init
-    // -------------------
-    navigator.presentation.defaultRequest = new PresentationRequest(presentationUrls);
+    // -----------
+    // DOM Element
+    // -----------
+    var button = document.getElementById('notsupported'),
+        notice = document.getElementById('notice');
 
     // ------------------------------
     // Start New Presentation with
     // 'default request' Test - BEGIN
     // ------------------------------
     async_test(function(t) {
-        navigator.presentation.defaultRequest.onconnectionavailable = t.step_func_done(function (evt) {
-            var connection = evt.connection;
+        var request = new PresentationRequest(presentationUrls);
+        if('defaultRequest' in navigator.presentation) {
+            navigator.presentation.defaultRequest = request;
 
-            assert_equals(connection.state, 'connecting', 'The initial state of the presentation connection is "connecting".');
-            assert_true(!connection.id, 'The connection ID is set.');
-            assert_true(typeof connection.id === 'string', 'The connection ID is a string.');
-            assert_true(connection instanceof PresentationConnection, 'The connection is an instance of PresentationConnection.');
-        });
-        document.getElementById('notsupported').onclick = t.step_func_done(function() {
+            if('onconnectionavailable' in request) {
+                request.onconnectionavailable = t.step_func_done(function (evt) {
+                    var connection = evt.connection;
+                    notice.parentNode.removeChild(notice);
+
+                    assert_equals(connection.state, 'connecting', 'The initial state of the presentation connection is "connecting".');
+                    assert_true(!connection.id, 'The connection ID is set.');
+                    assert_true(typeof connection.id === 'string', 'The connection ID is a string.');
+                    assert_true(connection instanceof PresentationConnection, 'The connection is an instance of PresentationConnection.');
+                });
+                button.onclick = t.step_func_done(function() {
+                    button.disabled = true;
+                    assert_unreached('This browser does not support defaultRequest.');
+                });
+            }
+            else {
+                notice.parentNode.removeChild(notice);
+                assert_unreached('This browser does not support connectionavailable event.');
+            }
+        }
+        else {
+            notice.parentNode.removeChild(notice);
             assert_unreached('This browser does not support defaultRequest.');
-        });
+        }
     }, '[Optional] The presentation was started successfully.');
```

Comment:
The test label is not very descriptive. I would drop it from here and update the `<title>` tag to something like "[Optional] Starting a presentation from the browser using a default presentation request"